src/api/FileCollaborators.js

```diff
@@ -114,6 +117,7 @@ class FileCollaborators extends MarkerBasedAPI {
         this.getFileCollaborators(fileId, successCallback, errorCallback, {
             filter_term: searchStr,
             include_groups: includeGroups,
+            respect_hidden_collabs: respectHiddenCollabs,
             include_uploader_collabs: false,
```

Comment:
nit: this looks should it be alphasorted
```suggestion
            include_groups: includeGroups,
            include_uploader_collabs: false,
            respect_hidden_collabs: respectHiddenCollabs,
```